providers/fab/pyproject.toml

```diff
@@ -57,7 +57,7 @@ license-files = ["NOTICE", "*/LICENSE*"]
 # Make sure to run ``breeze static-checks --type update-providers-dependencies --all-files``
 # After you modify the dependencies, and rebuild your Breeze CI image with ``breeze ci-image build``
 dependencies = [
-    "apache-airflow>=3.0.0",
+    "apache-airflow>=3.0.2",
```

Comment:
I am not sure this is needed? You can use the new FAB version with Airflow 3.0.0

Reply:
I added it because of the 4th point in:
https://github.com/apache/airflow/pull/50269#issuecomment-2871374182

I think it's safer to have it.

Reply:
Alright